static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -30,7 +34,11 @@ define([
                 var oldHref = link.attr('href');
                 var props = {
                     page: window.location,
-                    width: window.innerWidth
+                    width: window.innerWidth,
```

Comment:
I can. I had more parameters in the object before but I've deleted it so yes, the map is better now.